### ISSUE TITLE
Several nodes not working since docker update v2.05.70

### ISSUE THREAD

**Issue by yvessa** [dresden-elektronik/deconz-rest-plugin#2076]
Hello,

Since last update v2.05.70, a lot of nodes are not reporting motion presence. I tried to downgrade but same issue. I'm using a docker image.

Thank you in advance for your help.

Here is the beginning of the log showing several errors

[marthoc/deconz] Starting deCONZ...
[marthoc/deconz] Current deCONZ version: 2.05.70
[marthoc/deconz] Web UI port: 8080
[marthoc/deconz] Websockets port: 8443
[marthoc/deconz] VNC port: 6900
/usr/bin/xauth:  file /root/.Xauthority does not exist

New 'ryzen7:1000 (root)' desktop at :1000 on machine ryzen7

Starting applications specified in /etc/X11/Xvnc-session
Log file is /root/.vnc/ryzen7:1000.log

Use xtigervncviewer -SecurityTypes VncAuth,TLSVnc -passwd /root/.vnc/passwd ryzen7:1000 to connect to the VNC server.

libEGL warning: DRI2: failed to open swrast (search paths /usr/lib/x86_64-linux-gnu/dri:${ORIGIN}/dri:/usr/lib/dri)
libEGL warning: DRI2: failed to open swrast (search paths /usr/lib/x86_64-linux-gnu/dri:${ORIGIN}/dri:/usr/lib/dri)
libpng warning: iCCP: known incorrect sRGB profile
18:31:22:928 HTTP Server listen on address 0.0.0.0, port: 8080, root: /usr/share/deCONZ/webapp/
18:31:22:934 CTRL. 3.16.218:31:22:953 COM: /dev/ttyACM1 / serialno: 
18:31:22:953 COM: --dev: /dev/ttyACM1 (ConBee II)
18:31:22:953 ZCLDB init file /root/.local/share/dresden-elektronik/deCONZ/zcldb.txt
18:31:22:977 parent process /bin/sh
18:31:22:977 gw run mode: docker
18:31:22:977 GW sd-card image version file does not exist: /root/.local/share/dresden-elektronik/deCONZ/gw-version
18:31:22:977 DB sqlite version 3.16.2
18:31:22:977 DB PRAGMA page_count: 48
18:31:22:977 DB PRAGMA page_size: 4096
18:31:22:977 DB PRAGMA freelist_count: 3
18:31:22:977 DB file size 196608 bytes, free pages 3
18:31:22:977 DB PRAGMA user_version: 6
18:31:22:977 DB cleanup
18:31:22:977 DB create temporary views
18:31:22:986 don't close database yet, keep open for 900 seconds
18:31:22:987 started websocket server at port 8443
18:31:22:988 found node plugin: libde_rest_plugin.so - REST API Plugin
18:31:22:989 found node plugin: libde_signal_plugin.so - Signal Monitor Plugin
18:31:22:991 found node plugin: libstd_otau_plugin.so - STD OTAU Plugin
18:31:22:999 COM: /dev/ttyACM1 / serialno: 
18:31:22:999 COM: --dev: /dev/ttyACM1 (ConBee II)
18:31:23:055 Device firmware version 0x26490700
18:31:23:059 unlocked max nodes: 200
18:31:23:131 Device protocol version: 0x010B
18:31:23:138 new node - ext: 0x00212effff042b0e, nwk: 0x0000
18:31:23:200 don't close database yet, keep open for 900 seconds
18:31:23:201 LightNode 1: IkeaSwitch Sous-Sol Calo added
18:31:23:212 don't close database yet, keep open for 900 seconds
18:31:23:212 LightNode 2: IkeaSwitch Corridor added
18:31:23:217 don't close database yet, keep open for 900 seconds
18:31:23:217 LightNode 3: IkeaSwitch Salle Manger added
18:31:23:223 don't close database yet, keep open for 900 seconds
18:31:23:223 LightNode 4: IkeaSwitch Chambre Maitres added
18:31:23:228 don't close database yet, keep open for 900 seconds
18:31:23:228 LightNode 5: IkeaSwitch Petite Chambre Haut added
18:31:23:233 don't close database yet, keep open for 900 seconds
18:31:23:234 LightNode 6: IkeaSwitch Atelier added
18:31:23:239 don't close database yet, keep open for 900 seconds
18:31:23:239 LightNode 7: IkeaSwitch Petite Chambre Bas added
18:31:23:245 don't close database yet, keep open for 900 seconds
18:31:23:245 LightNode 8: IkeaSwitch Corridor Ext added
18:31:23:250 don't close database yet, keep open for 900 seconds
18:31:23:250 LightNode 9: IkeaSwitch Garage Etabli added
18:31:23:256 don't close database yet, keep open for 900 seconds
18:31:23:256 LightNode 10: IkeaSwitch Garage Evier added
18:31:23:262 don't close database yet, keep open for 900 seconds
18:31:23:262 LightNode 11: IkeaSwitch Remise added
18:31:23:267 SensorNode 2 set node 0x00158d0002722b8c
18:31:23:273 SensorNode 3 set node 0x00158d0002c9c8ff
18:31:23:278 SensorNode 4 set node 0x00158d00027378bb
18:31:23:283 SensorNode 5 set node 0x00158d0002751a7e
18:31:23:290 SensorNode 6 set node 0x00158d0002725f46
18:31:23:296 SensorNode 7 set node 0x00158d0002bfdbd8
18:31:23:300 SensorNode 8 set node 0x00158d0002e230c9
18:31:23:308 SensorNode 9 set node 0x00158d0002bfc412
18:31:23:313 SensorNode 10 set node 0x00158d0002e9230d
18:31:23:313 SensorNode 11 set node 0x00158d0002e9230d
18:31:23:318 SensorNode 12 set node 0x00158d000272be37
18:31:23:319 SensorNode 13 set node 0x00158d000272be37
18:31:23:324 SensorNode 14 set node 0x00158d000257f5a0
18:31:23:324 SensorNode 15 set node 0x00158d000257f5a0
18:31:23:329 SensorNode 16 set node 0x00158d0002d43c7c
18:31:23:329 SensorNode 17 set node 0x00158d0002d43c7c
18:31:23:334 SensorNode 18 set node 0x00158d00029c01f7
18:31:23:334 SensorNode 19 set node 0x00158d00029c01f7
18:31:23:339 SensorNode 20 set node 0x00158d00029c0a58
18:31:23:339 SensorNode 21 set node 0x00158d00029c0a58
18:31:23:343 SensorNode 22 set node 0x00158d000257f448
18:31:23:343 SensorNode 23 set node 0x00158d000257f448
18:31:23:348 SensorNode 24 set node 0x00158d0002584686
18:31:23:348 SensorNode 25 set node 0x00158d0002584686
18:31:23:353 SensorNode 26 set node 0x00158d000257f42f
18:31:23:353 SensorNode 62 set node 0x00158d000257f42f
18:31:23:358 SensorNode 27 set node 0x00158d0002d43bf6
18:31:23:358 SensorNode 28 set node 0x00158d0002d43bf6
18:31:23:363 SensorNode 29 set node 0x00158d00029c0251
18:31:23:363 SensorNode 30 set node 0x00158d00029c0251
18:31:23:367 SensorNode 31 set node 0x00158d0002732686
18:31:23:368 SensorNode 32 set node 0x00158d0002732686
18:31:23:375 SensorNode 33 set node 0x00158d00029c018c
18:31:23:375 SensorNode 34 set node 0x00158d00029c018c
18:31:23:380 SensorNode 35 set node 0x00158d000273266c
18:31:23:380 SensorNode 36 set node 0x00158d000273266c
18:31:23:385 SensorNode 37 set node 0x00158d00027327b9
18:31:23:385 SensorNode 38 set node 0x00158d00027327b9
18:31:23:390 SensorNode 39 set node 0x00158d00029a97eb
18:31:23:390 SensorNode 40 set node 0x00158d00029a97eb
18:31:23:395 SensorNode 41 set node 0x00158d0002752138
18:31:23:395 SensorNode 42 set node 0x00158d0002752138
18:31:23:399 SensorNode 43 set node 0x00158d000275225c
18:31:23:400 SensorNode 44 set node 0x00158d000275225c
18:31:23:404 SensorNode 45 set node 0x00158d000257f724
18:31:23:404 SensorNode 46 set node 0x00158d000257f724
18:31:23:413 SensorNode 47 set node 0x00158d0002583a0f
18:31:23:413 SensorNode 48 set node 0x00158d0002583a0f
18:31:23:418 SensorNode 49 set node 0x00158d0002774473
18:31:23:426 SensorNode 50 set node 0x00158d00026e26ad
18:31:23:431 SensorNode 51 set node 0x00158d00022f00ac
18:31:23:436 SensorNode 52 set node 0x00158d00024bcb52
18:31:23:441 SensorNode 53 set node 0x00158d0002773efb
18:31:23:448 SensorNode 54 set node 0x00158d0002773f01
18:31:23:454 SensorNode 55 set node 0x00158d0002702c2d
18:31:23:458 SensorNode 56 set node 0x00158d0002566d9b
18:31:23:463 SensorNode 57 set node 0x00158d00023347a0
18:31:23:468 SensorNode 58 set node 0x00158d000277447d
18:31:23:473 SensorNode 59 set node 0x00158d0002773ef1
18:31:23:478 SensorNode 60 set node 0x00158d0002401657
18:31:23:482 SensorNode 61 set node 0x00158d00028076bc
18:31:23:487 SensorNode 63 set node 0x00158d00027adefb
18:31:23:492 SensorNode 64 set node 0x000b57fffeaf68e8
18:31:23:497 SensorNode 65 set node 0x000b57fffea72c66
18:31:23:504 SensorNode 66 set node 0x000b57fffea6cc9d
18:31:23:509 SensorNode 67 set node 0x000b57fffead1185
18:31:23:514 SensorNode 68 set node 0x00158d0002ca0620
18:31:23:514 SensorNode 69 set node 0x00158d0002ca0620
18:31:23:514 SensorNode 70 set node 0x00158d0002ca0620
18:31:23:519 SensorNode 74 set node 0x00158d0002c8ee7c
18:31:23:519 SensorNode 75 set node 0x00158d0002c8ee7c
18:31:23:519 SensorNode 76 set node 0x00158d0002c8ee7c
18:31:23:524 SensorNode 80 set node 0x00158d0002c8eca3
18:31:23:524 SensorNode 81 set node 0x00158d0002c8eca3
18:31:23:524 SensorNode 82 set node 0x00158d0002c8eca3
18:31:23:528 SensorNode 89 set node 0x00158d0002e954cb
18:31:23:529 SensorNode 91 set node 0x00158d0002e954cb
18:31:23:529 SensorNode 90 set node 0x00158d0002e954cb
18:31:23:533 SensorNode 92 set node 0x00158d00025d9109
18:31:23:533 SensorNode 93 set node 0x00158d00025d9109
18:31:23:533 SensorNode 94 set node 0x00158d00025d9109
18:31:23:538 SensorNode 95 set node 0x00158d0002e98d2c
18:31:23:538 SensorNode 96 set node 0x00158d0002e98d2c
18:31:23:538 SensorNode 97 set node 0x00158d0002e98d2c
18:31:23:543 SensorNode 86 set node 0x00158d0002741c1c
18:31:23:543 SensorNode 87 set node 0x00158d0002741c1c
18:31:23:543 SensorNode 88 set node 0x00158d0002741c1c
18:31:23:547 SensorNode 101 set node 0x00158d0002d43f0e
18:31:23:548 SensorNode 102 set node 0x00158d0002d43f0e
18:31:23:552 SensorNode 103 set node 0x00158d0002c13bd9
18:31:23:552 SensorNode 104 set node 0x00158d0002c13bd9
18:31:23:552 SensorNode 105 set node 0x00158d0002c13bd9
18:31:23:557 SensorNode 106 set node 0x00158d00025efa1d
18:31:23:557 SensorNode 107 set node 0x00158d00025efa1d
18:31:23:557 SensorNode 108 set node 0x00158d00025efa1d
18:31:23:571 SensorNode 77 set node 0x00158d0002b5b75e
18:31:23:571 SensorNode 78 set node 0x00158d0002b5b75e
18:31:23:571 SensorNode 79 set node 0x00158d0002b5b75e
18:31:23:572 SensorNode 98 set node 0x00158d0002c14bfc
18:31:23:572 SensorNode 99 set node 0x00158d0002c14bfc
18:31:23:572 SensorNode 100 set node 0x00158d0002c14bfc
18:31:23:627 Current channel 15
18:31:23:643 CTRL ANT_CTRL 0x03
18:31:23:667 Device protocol version: 0x010B
18:31:23:715 Current channel 15
18:31:23:731 CTRL ANT_CTRL 0x03
18:31:23:767 no button map for: lumi.sensor_motion.aq2 ep: 0x01 cl: 0x0000 cmd: 0x0A pl[0]: 001
18:31:23:767 ZCL attribute report 0x00158D0002752138 for cluster 0x0000, ep 0x01
18:31:23:767 0x00158D0002752138 extract Xiaomi special attribute 0xFF01
18:31:23:767 	01 battery 3005 (0x0BBD)
18:31:23:767 	03 temperature 27 °C
18:31:23:767 	04 unknown 17320 (0x43A8)
18:31:23:767 	05 RSSI dB (?) 20 (0x0014)
18:31:23:767 	06 LQI (?) 30064771072 (0x0700000000)
18:31:23:767 	0a unknown 0 (0x0000)
18:31:23:767 	64 on/off 0
18:31:23:767 	0b lightlevel 42 (0x002A)
18:31:26:696 New websocket 192.168.1.105:38206 (state: 3) 
18:31:26:808 drop request to zombie (rx = 0)
18:31:28:033 don't close database yet, keep open for 900 seconds
18:31:28:042 GW update firmware found: /usr/share/deCONZ/firmware/deCONZ_ConBeeII_0x26490700.bin.GCF
18:31:28:042 GW firmware version: 0x26490700
18:31:28:042 GW firmware version is up to date: 0x26490700
18:31:28:734 Announced to internet
18:31:29:082 Remove websocket 192.168.1.105:38206 after error Unknown error
18:31:33:676 New websocket 192.168.1.105:38560 (state: 3) 
18:31:33:906 no button map for: lumi.sensor_motion.aq2 ep: 0x01 cl: 0x0400 cmd: 0x0A pl[0]: 000
18:31:33:906 ZCL attribute report 0x00158D0002583A0F for cluster 0x0400, ep 0x01
18:31:47:122 Bind response success for 0x90fd9ffffe336489 cluster 0x0000
18:31:47:234 ZCL configure reporting rsp seq: 38 0x90FD9FFFFE336489 for cluster 0x0000 attr 0x4000 status 0x00
18:31:47:336 Bind response success for 0x90fd9ffffe336489 cluster 0x0006
18:31:47:447 ZCL configure reporting rsp seq: 39 0x90FD9FFFFE336489 for cluster 0x0006 attr 0x0000 status 0x00
18:31:47:542 Bind response success for 0x90fd9ffffe336489 cluster 0x0008
18:31:47:542 skip configure report for cluster: 0x0008 attr: 0x0000 of node 0x90FD9FFFFE336489 (wait reading or unsupported)
18:31:47:972 APS-DATA.indication from unknown node 0xD028
18:31:47:972 ZCL attribute report 0x0000000000000000 for cluster 0x0402, ep 0x01
18:31:54:876 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:31:54:876 ZCL attribute report 0x00158D0002CA0620 for cluster 0x0402, ep 0x01
18:31:54:885 no button map for: lumi.weather ep: 0x01 cl: 0x0405 cmd: 0x0A pl[0]: 000
18:31:54:885 ZCL attribute report 0x00158D0002CA0620 for cluster 0x0405, ep 0x01
18:31:54:889 no button map for: lumi.weather ep: 0x01 cl: 0x0403 cmd: 0x0A pl[0]: 000
18:31:54:889 ZCL attribute report 0x00158D0002CA0620 for cluster 0x0403, ep 0x01
18:32:01:740 0xD0CF5EFFFE7B85A4 error APSDE-DATA.confirm: 0xD0 on task
18:32:06:632 Bind response success for 0x90fd9ffffe336489 cluster 0x0000
18:32:08:644 Bind response success for 0x90fd9ffffe336489 cluster 0x0006
18:32:08:745 ZCL configure reporting rsp seq: 50 0x90FD9FFFFE336489 for cluster 0x0006 attr 0x0000 status 0x00
18:32:08:988 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:32:08:988 ZCL attribute report 0x00158D0002741C1C for cluster 0x0402, ep 0x01
18:32:13:819 scan finished
18:32:19:834 no button map for: lumi.sensor_motion.aq2 ep: 0x01 cl: 0x0400 cmd: 0x0A pl[0]: 000
18:32:19:834 ZCL attribute report 0x00158D000257F724 for cluster 0x0400, ep 0x01
18:32:23:032 Current channel 15
18:32:23:040 Device TTL 4827 s flags: 0x7
18:32:28:040 0xD0CF5EFFFE7B85A4 error APSDE-DATA.confirm: 0xD0 on task
18:32:32:935 no button map for: lumi.sensor_wleak.aq1 ep: 0x01 cl: 0x0000 cmd: 0x0A pl[0]: 001
18:32:32:935 ZCL attribute report 0x00158D0002773F01 for cluster 0x0000, ep 0x01
18:32:32:935 0x00158D0002773F01 extract Xiaomi special attribute 0xFF01
18:32:32:935 	01 battery 3015 (0x0BC7)
18:32:32:935 	03 temperature 24 °C
18:32:32:935 	04 unknown 17320 (0x43A8)
18:32:32:935 	05 RSSI dB (?) 8 (0x0008)
18:32:32:935 	06 LQI (?) 17180065792 (0x0400030000)
18:32:32:935 	08 unknown 516 (0x0204)
18:32:32:935 	0a unknown 0 (0x0000)
18:32:32:935 	64 on/off 0
18:32:34:390 no button map for: lumi.sensor_motion.aq2 ep: 0x01 cl: 0x0400 cmd: 0x0A pl[0]: 000
18:32:34:390 ZCL attribute report 0x00158D0002583A0F for cluster 0x0400, ep 0x01
18:32:41:231 max transmit errors for node 0xD0CF5EFFFECB2910, last seen by neighbors 2 s
18:32:49:475 ZCL attribute report 0xD0CF5EFFFE7B8BB1 for cluster 0x0000, ep 0x01
18:32:49:535 ZCL attribute report 0xD0CF5EFFFE7B8BB1 for cluster 0x0000, ep 0x01
18:33:21:823 no button map for: lumi.sensor_motion.aq2 ep: 0x01 cl: 0x0400 cmd: 0x0A pl[0]: 000
18:33:21:823 ZCL attribute report 0x00158D000257F724 for cluster 0x0400, ep 0x01
18:33:22:904 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:33:22:904 ZCL attribute report 0x00158D0002C8ECA3 for cluster 0x0402, ep 0x01
18:33:22:914 no button map for: lumi.weather ep: 0x01 cl: 0x0405 cmd: 0x0A pl[0]: 000
18:33:22:914 ZCL attribute report 0x00158D0002C8ECA3 for cluster 0x0405, ep 0x01
18:33:22:918 no button map for: lumi.weather ep: 0x01 cl: 0x0403 cmd: 0x0A pl[0]: 000
18:33:22:918 ZCL attribute report 0x00158D0002C8ECA3 for cluster 0x0403, ep 0x01
18:33:23:032 Current channel 15
18:33:23:040 Device TTL 4767 s flags: 0x7
18:33:30:447 max transmit errors for node 0xD0CF5EFFFECB1D62, last seen by neighbors 42 s
18:33:30:856 verified group capacity: 255 and group count: 1 of LightNode 0x90fd9ffffe336489
18:33:30:856 0x90fd9ffffe336489 found group 0xFFF0
18:33:30:995 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:33:30:995 ZCL attribute report 0x00158D0002E954CB for cluster 0x0402, ep 0x01
18:33:31:612 verified group capacity: 255 and group count: 1 of LightNode 0xd0cf5efffecb2910
18:33:31:612 0xd0cf5efffecb2910 found group 0xFFF0
18:33:32:347 verified group capacity: 255 and group count: 1 of LightNode 0xd0cf5efffedcd9db
18:33:32:347 0xd0cf5efffedcd9db found group 0xFFF0
18:33:33:110 verified group capacity: 255 and group count: 1 of LightNode 0xd0cf5efffecb1d62
18:33:33:110 0xd0cf5efffecb1d62 found group 0xFFF0
18:33:33:877 verified group capacity: 255 and group count: 1 of LightNode 0xd0cf5efffe7cc69e
18:33:33:877 0xd0cf5efffe7cc69e found group 0xFFF0
18:33:34:612 verified group capacity: 255 and group count: 1 of LightNode 0x90fd9ffffe2bc4b5
18:33:34:612 0x90fd9ffffe2bc4b5 found group 0xFFF0
18:33:35:361 verified group capacity: 255 and group count: 1 of LightNode 0xd0cf5efffe7b8bb1
18:33:35:361 0xd0cf5efffe7b8bb1 found group 0xFFF0
18:33:37:567 verified group capacity: 255 and group count: 1 of LightNode 0x90fd9ffffe336592
18:33:37:567 0x90fd9ffffe336592 found group 0xFFF0
18:33:40:834 0xD0CF5EFFFECB2910 error APSDE-DATA.confirm: 0xA7 on task
18:33:41:961 0xD0CF5EFFFECB1D62 error APSDE-DATA.confirm: 0xA7 on task
18:33:43:757 0x90FD9FFFFE2BC4B5 error APSDE-DATA.confirm: 0xA7 on task
18:33:53:845 0xD0CF5EFFFE7B85A4 error APSDE-DATA.confirm: 0xD0 on task
18:34:00:633 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:34:00:633 ZCL attribute report 0x00158D0002C13BD9 for cluster 0x0402, ep 0x01
18:34:00:646 no button map for: lumi.weather ep: 0x01 cl: 0x0405 cmd: 0x0A pl[0]: 000
18:34:00:646 ZCL attribute report 0x00158D0002C13BD9 for cluster 0x0405, ep 0x01
18:34:00:660 no button map for: lumi.weather ep: 0x01 cl: 0x0403 cmd: 0x0A pl[0]: 000
18:34:00:660 ZCL attribute report 0x00158D0002C13BD9 for cluster 0x0403, ep 0x01
18:34:11:886 no button map for: lumi.sensor_wleak.aq1 ep: 0x01 cl: 0x0000 cmd: 0x0A pl[0]: 001
18:34:11:886 ZCL attribute report 0x00158D0002566D9B for cluster 0x0000, ep 0x01
18:34:11:886 0x00158D0002566D9B extract Xiaomi special attribute 0xFF01
18:34:11:886 	01 battery 3025 (0x0BD1)
18:34:11:886 	03 temperature 19 °C
18:34:11:886 	04 unknown 17320 (0x43A8)
18:34:11:886 	05 RSSI dB (?) 33 (0x0021)
18:34:11:886 	06 LQI (?) 4294967296 (0x0100000000)
18:34:11:886 	08 unknown 516 (0x0204)
18:34:11:886 	0a unknown 0 (0x0000)
18:34:11:886 	64 on/off 0
18:34:14:559 0xD0CF5EFFFE7B85A4 error APSDE-DATA.confirm: 0xD0 on task
18:34:23:033 Current channel 15
18:34:23:041 Device TTL 4707 s flags: 0x7
18:34:29:363 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:34:29:363 ZCL attribute report 0x00158D00025EFA1D for cluster 0x0402, ep 0x01
18:34:29:371 no button map for: lumi.weather ep: 0x01 cl: 0x0405 cmd: 0x0A pl[0]: 000
18:34:29:371 ZCL attribute report 0x00158D00025EFA1D for cluster 0x0405, ep 0x01
18:34:29:397 no button map for: lumi.weather ep: 0x01 cl: 0x0403 cmd: 0x0A pl[0]: 000
18:34:29:397 ZCL attribute report 0x00158D00025EFA1D for cluster 0x0403, ep 0x01
18:34:35:883 no button map for: lumi.sensor_motion.aq2 ep: 0x01 cl: 0x0400 cmd: 0x0A pl[0]: 000
18:34:35:883 ZCL attribute report 0x00158D0002583A0F for cluster 0x0400, ep 0x01
18:34:37:691 no button map for: lumi.weather ep: 0x01 cl: 0x0402 cmd: 0x0A pl[0]: 000
18:34:37:691 ZCL attribute report 0x00158D00025D9109 for cluster 0x0402, ep 0x01
18:34:37:700 no button map for: lumi.weather ep: 0x01 cl: 0x0405 cmd: 0x0A pl[0]: 000
18:34:37:700 ZCL attribute report 0x00158D00025D9109 for cluster 0x0405, ep 0x01
18:34:37:704 no button map for: lumi.weather ep: 0x01 cl: 0x0403 cmd: 0x0A pl[0]: 000
18:34:37:704 ZCL attribute report 0x00158D00025D9109 for cluster 0x0403, ep 0x01
18:34:39:339 APS-DATA.indication from unknown node 0xB359
18:34:39:339 ZCL attribute report 0x0000000000000000 for cluster 0x0402, ep 0x01
18:34:41:056 0xD0CF5EFFFE7B85A4 error APSDE-DATA.confirm: 0xD0 on task
18:34:47:927 no button map for: lumi.sensor_wleak.aq1 ep: 0x01 cl: 0x0000 cmd: 0x0A pl[0]: 001
18:34:47:927 ZCL attribute report 0x00158D00023347A0 for cluster 0x0000, ep 0x01
18:34:47:927 0x00158D00023347A0 extract Xiaomi special attribute 0xFF01
18:34:47:927 	01 battery 3045 (0x0BE5)
18:34:47:927 	03 temperature 23 °C
18:34:47:927 	04 unknown 17320 (0x43A8)
18:34:47:927 	05 RSSI dB (?) 42 (0x002A)
18:34:47:927 	06 LQI (?) 4294967296 (0x0100000000)
18:34:47:927 	08 unknown 516 (0x0204)
18:34:47:927 	0a unknown 0 (0x0000)
18:34:47:927 	64 on/off 1
18:34:50:605 max transmit errors for node 0xD0CF5EFFFEDCD9DB, last seen by neighbors 122 s
18:34:59:615 Bind response success for 0xd0cf5efffe7b8bb1 cluster 0x0000

**Comment by yvessa** [comment]
Not sure it's a v2.05.70 issue. Anyhow after reading there is some issues with it, I stay with v2.05.69. 
 
I have noticed some routers (ikea switches) were in gray, I unplugged and plugged back these. Then, few sensors are now working. Let's wait few hours if network rebuild by itself.

Is there a troubleshooting page about these kind of issues? I do not want to start from scratch, I have like 70 nodes... Some hints might be helpful avoiding clogging forum. I might not have found the good location yet.

Thanks

**Comment by yvessa** [comment]
Forgot to ask, is there a way to know which parent (ikea switch) is allocated to a child (sensors)? I remember reading in the past than xiaomi sensors are locked to a parent. That might explain some issues I have. It would be nice to know parent/child association to diagnose.

**Comment by ebaauw** [comment]
If you're lucky, the deCONZ GUI might show a line connecting the end device to its parent.  Note that these lines are maintained by querying the neighbour tables on the devices and might not reflect the actual situation.

Otherwise: sniff the ZigBee traffic and see at MAC level to which router the device sends its messages.

**Comment by yvessa** [comment]
Thanks ebaauw, most were not showing a line. ZigBee sniffing would definitely help. It would be nice if we could reallocate a node to a specific parent.

In order to get rid of these issues, I shutdown deconz completely. Then unplug and plug back router nodes (ikea switches). When I started deconz everything was back to normal. Also, updated to v2.05.70. At least I did not have to re-add all my sensors.

I tried unplugging/plugging routers w/o shutting down deconz and it did not work. 

I can't explain what happened exactly.

**Comment by peer69** [comment]
I tried that as well. Even unplugged the Conbee Stick as I didnt want to shut down the whole ESXi Host after the deconz vm was shut down. I will probably now set up a test machine with some of the devices as I have a second adapter but I obviously wont be able to replicate my productive environment so I dont know if this might prevent similar scenarios with further updates at all. 
It really felt like this update caused the coordinator to drop all routers and sensors connected through routers from the network so this might be neighbor table related. I moved sensors formerly connected through routers near the gateway but this did nothing. Only sensors connected directly to the gateway before the update were reachable afterwards.
